.github/workflows/haskell.yml

```diff
@@ -230,6 +230,8 @@ jobs:
 
       SECP_CACHE_VERSION: "2022-12-30"
 
+      packages-with-ruby-cddl-tests: fromJson('["cardano-protocol-tpraos", "cardano-ledger-shelley", "cardano-ledger-allegra", "cardano-ledger-mary", "cardano-ledger-alonzo", "cardano-ledger-babbage", "cardano-ledger-conway"]')
```

Comment:
Are you sure the `fromJson` is being executed in the `if:` expression? It could be just searching the JSON-encoded string, which isn't what you want since it would match prefixes.

You could make it more explicit by putting the `fromJson` call inside the `contains` check.

Reply:
It's a pity this array couldn't be encoded explicitly in YAML, but sadly GHA doesn't have native support for variables with structured values.

Reply:
> Are you sure the fromJson is being executed in the if: expression?

Yes, I am sure, otherwise CI would fail.

Reply:
I see what you mean. Yeah, it is likely the case that it is just searching through the string.
Just to be safe I'll move the use of `fromJson` as you suggested.

That's what happens when there are no types :grinning: